### PR TITLE
Correct so that relativeQuantity affects base units as well as DisplayUnits

### LINKED ISSUE #1165: FMI 3.0: Unit specification error regarding relativeQuantity
In Section _2.2.2. Definition of Units_ the conversion of units based on `<BaseUnit>` is described.

One important pitfall is to incorporate the `offset` if `relativeQuantity="true"`; an example is given in subsection _Signal connection check_ which even refers to the definition of `relativeQuantity` below.

The problem now is, that the definition/description of `relativeQuantity` only talks about semantic means w.r.t. `displayUnit`:

> If this attribute is true, then the offset of displayUnit must be ignored. [For example, 10 degree Celsius = 10 Kelvin if relativeQuantity = true and not 283.15 Kelvin.]

I think, this is wrong. The `relativeQuantity` is much more important for the formal conversion of units based on `<BaseUnit>` definitions.

The definition of `relativeQuantity` should be respectively adapted. Also a more complex example how to use it for unit conversions has to be added.

Finally, the formula given for `<BaseUnit>`-based unit conversion should already incorporate `relativeQuantity`. At the moment it is just:

 > BaseUnit_value = factor * Unit_value + offset

but I guess it should be `BaseUnit_value = factor * Unit_value + (if relativeQuantity 0.0 else offset)`, although it now requires some linking to `relativeQuantity` which is not part of the `<Unit>`/`<BaseUnit>` definition itself but rather variable specific (it is an attribute of `<Float64Type>`, not `<BaseUnit>`).

### PR DESCRIPTION
**Pull request by KarlWernersson**
fix #1165

### CODE REVIEW

**Review by t-sommer — APPROVED** [review]

File: docs/2_4_common_schema.adoc [hunk 1]
```diff
@@ -346,7 +346,7 @@ A value latexmath:[v_{unit}] in `Unit` is converted to the base unit latexmath:[
 
 [latexmath]
 ++++
-v_{base} = \texttt{factor} * v_{unit} + \texttt{offset}
+v_{base} = \texttt{factor} * v_{unit} + \texttt{if relativeQuantity then 0 else offset}
```
Comment (chrbertsch) [inline docs/2_4_common_schema.adoc:349]: Should a  similar adaptation not be explicily made for
_(in this case the relativeQuantity of the <TypeDefinition>, see below, has to be taken into account in order to determine whether offset shall or shall not be utilized):_
`factor(v1) * v1 + offset(v1) = factor(v2) * v2 + offset(v2)`
in the paragraph "Unit check when connecting variables of different FMUs"

Shouldn't we explain, that `factor `and `offset ` are properties of the BaseUnit, while `relativeQuantity` is an attribute of a type and link to its definition

Comment (KarlWernersson) [reply docs/2_4_common_schema.adoc:349]: Yes that would be better

Comment (chrbertsch) [reply docs/2_4_common_schema.adoc:349]: @t-sommer : in which order shall we proceed?

**Review by chrbertsch — APPROVED** [review]

**Comment by chrbertsch** [comment]
Can we merge this? Please review. I would like to  port this to FMI 2.0.2

**Review by t-sommer — APPROVED** [review]

**Pull request merged**